unittests/Swift/BuildSystemBindingsTests.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  BuildSystemBindingsTests.swift
+//  llbuildSwiftTests
+//
+//  Copyright © 2019 Apple Inc. All rights reserved.
+//
+
+import XCTest
+
+// The Swift package has llbuildSwift as module
+#if SWIFT_PACKAGE
+import llbuild
+@testable import llbuildSwift
+#else
+@testable import llbuild
+#endif
+
+extension Command {
+  func with(name: String? = nil, shouldShowStatus: Bool? = nil, description: String? = nil, verboseDescription: String? = nil) -> Command {
+    return Command(name: name ?? self.name, shouldShowStatus: shouldShowStatus ?? self.shouldShowStatus, description: description ?? self.description, verboseDescription: verboseDescription ?? self.verboseDescription)
+  }
+}
+
+class BuildSystemBindingsTests: XCTestCase {
```

Comment:
Should we run these tests on Linux as well?

Reply:
We should! I'll generate a LinuxMain

Reply:
Done

Reply:
We don't use SwiftPM to build llbuild on SwiftCI so LinuxMain won't really help..

Reply:
What are we then using? Or where do I need to add the tests?

Reply:
Swift-CI directly runs the CMake build (with Ninja) and the Xcode project.